website/content/en/docs/deploy-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Development Guide"
```

Comment:
Can we call this Operational Requirements?

Reply:
Guessing you didn't mean to commit this.

Reply:
yes, this has been removed from the PR :)